Source/CarthageKit/Project.swift

```diff
@@ -313,15 +313,18 @@ public final class Project {
 			}
 	}
 	
-	/// Loads the Cartfile for the given dependency, at the given version.
-	private func cartfileForDependency(dependency: Dependency<PinnedVersion>) -> SignalProducer<Cartfile, CarthageError> {
+	/// Loads the dependencies for the given dependency, at the given version.
+	private func dependencies(for dependency: Dependency<PinnedVersion>) -> SignalProducer<Dependency<VersionSpecifier>, CarthageError> {
```

Comment:
I tend to prefer it this way—it's often easier to work with and it's also easy to `collect`.